xml/System.Diagnostics/DiagnosticListener.xml

```diff
@@ -161,8 +161,8 @@
       </Parameters>
       <Docs>
         <param name="name">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Override abstract method. Meant to check if the DiagnosticListener is enabled.</summary>
```

Comment:
```suggestion
        <summary>Checks whether the <see cref="T:System.Diagnostics.DiagnosticListener" /> is enabled.</summary>
```
But I'd consider just deleting the documentation here. The alternative is to document the `name` parameter and provide a summary that indicates what this method does that differentiates it from `Enabled()`.

Reply:
I think we can keep this one, unless @wtgodbe thinks differently.